ReactAndroid/src/main/java/com/facebook/react/modules/storage/AsyncStorageModule.java

```diff
@@ -83,9 +85,9 @@ public void multiGet(final ReadableArray keys, final Callback callback) {
       return;
     }
 
-    new GuardedAsyncTask<Void, Void>(getReactApplicationContext()) {
+    execute(new Runnable() {
```

Comment:
What happens if an exception is thrown when executing the Runnable?

GuardedAsyncTask provided error management, see: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/GuardedAsyncTask.java#L34

Reply:
Could we use `executeOnExecutor` (https://developer.android.com/reference/android/os/AsyncTask.html#executeOnExecutor(java.util.concurrent.Executor,%20Params...)) instead of `execute` and keep using GuardedAsyncTask instead?

Reply:
@janicduplessis got it and ready to push the change, but waiting for CI to become green again.